globals/colours.css

```diff
@@ -9,7 +9,7 @@
   --color-black:        rgb(51,51, 51);
 
   --color-green:        rgb(41,174,155);
-  --color-red:          rgb(252,99,99);
+  --color-red:          var(--color-coral);
```

Comment:
The green and red in this case are application colours. If we wanted the green to be teal, I _would_ change the green variable to equal teal.

It means that changes to our brand colours propagate to our application colours without having to change individual references. It also means we can move application colours away from brand colours without potentially leaving references to the brand colours, or editing them.